docs/specs/idp-account-linking.md

```diff
@@ -12,47 +12,60 @@ Here is an example of the configuration:
 
 ```yaml
 identity:
-  providers:
-    - alias: google
-      client_id: exampleclientid
-      type: google
-      account_linking:
-        enabled: true
-        idp_claim_key: "email"
-        match_against_claim_key: "email"
+  on_conflict:
+    account_linking:
+      oauth: "merge"
+  oauth:
+    providers:
+      - alias: google
+        client_id: exampleclientid
+        type: google
+        account_linking:
+          enabled: false
```

Comment:
What does `enabled: false` mean? Remember we have to be back-forward compatible. That is, the default config should match the current behavior, that is, error in case of conflict.

If `enabled: false` means `account_linking.oauth=error`, it could be confusing.

Reply:
@louischan-oursky `enabled: false` means the oauth account will never be considered as matching an existing identity.

So I guess the default should be `enabled: true` with `email` for some of the provider, but some of them can still remain `enabled: false` as default such as wechat. Maybe let me list out the appropriate default values.

Reply:
```
link_by_claim: "email" # nullable
```